PayButton/paybutton-server
# 2  [Issue #395] feat: use user addresses on wallets

**Description:**
Using the new field `wallet.userAddresses` everywhere `wallet.addresses` was previously used.
Still does not solves  #426. Users are able to create add buttons with a shared address to their wallets; but they are gonna see both buttons (theirs and the one of the other user).

**Test plan:**
- Try to create a paybutton assigning it to wallet;
- Try to edit some wallet paybuttons;
- Try to create a new wallet with paybuttons;

All should work, with the aforementioned caveat.

Depends on:
- [x] #429 
- [x] #434
